android/gradle.properties

```diff
@@ -0,0 +1,2 @@
+android.enableJetifier=true
```

Comment:
This is not necessary since there are no other dependencies that this library depends on.

Reply:
Removed 👍